foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/FoundationCoreJavaAiopsReferencesMigration.java

```diff
@@ -0,0 +1,93 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.boozallen.aissemble.upgrade.util.FileUtils.replaceLiteralInFile;
+
+public class FoundationCoreJavaAiopsReferencesMigration extends AbstractAissembleMigration {
```

Comment:
Q: Can we not re-use/generalize the migration we already have for updating Java package names?

Reply:
This was a great idea! I just merged both of the classes into one called "AiopsReferenceMigration".